eos_data_distribution/ndn/file.py

```diff
@@ -274,6 +264,44 @@ def _on_complete(self):
         os.unlink(self._sgt_filename)
         super(FileConsumer, self)._on_complete()
 
+    def _create_files(self, filename):
+        self._filename = filename
+        mkdir_p(os.path.dirname(filename))
+        self._part_filename = '%s.part' % (filename, )
+        self._part_fd = os.open(
+            self._part_filename, os.O_CREAT | os.O_WRONLY | os.O_NONBLOCK, 0o600)
+        self._sgt_filename = '%s.sgt' % (filename, )
+        self._sgt_fd = os.open(
+            self._sgt_filename, os.O_CREAT | os.O_RDWR, 0o600)
+
+
+class FileConsumer(Consumer):
+
+    """Write to a File"""
```

Comment:
I don’t think that’s suitable as documentation. Either remove it (so sphinx says that `FileConsumer` needs documenting), or expand it. At the moment it’s too short to be useful, but will make `FileConsumer` disappear from the lists of things which need documenting.